clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/description/RegisterInstancesWithGoogleLoadBalancerDescription.groovy

```diff
@@ -16,9 +16,23 @@
 
 package com.netflix.spinnaker.clouddriver.google.deploy.description
 
-class RegisterInstancesWithGoogleLoadBalancerDescription extends AbstractGoogleCredentialsDescription {
+import com.netflix.frigga.Names
+import com.netflix.spinnaker.clouddriver.security.resources.ApplicationNameable
+
+class RegisterInstancesWithGoogleLoadBalancerDescription extends AbstractGoogleCredentialsDescription implements ApplicationNameable{
   List<String> loadBalancerNames
   List<String> instanceIds
   String region
   String accountName
+
+  @Override
+  Collection<String> getApplications() {
+    def list = (loadBalancerNames - null)
```

Comment:
fwiw the `DescriptionAuthorizer` should handle nulls but it's no doubt safer to be explicit

Reply:
Ack.